### PR TITLE
Fix the grammar in function examples of Beam3D

### PR DESCRIPTION
**Pull request by JebronLames32**
#### Brief description of what is fixed or changed
The example problems in most functions had a common grammatical error.
That has been changed in the respective functions. 
The list of functions where change is made are:
plot_shear_force, plot_bending_moment, plot_slope,plot_deflection, 
plot_loading_results, plot_shear_stress, max_shear_force, max_bending_moment,
max_deflection.

#### Other comments
NO ENTRY

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -2880,7 +2880,7 @@ def plot_shear_force(self, dir="all", subs=None):
         Examples
         ========
         There is a beam of length 20 meters. It it supported by rollers
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:2882]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 2]
```diff
@@ -2982,7 +2982,7 @@ def plot_bending_moment(self, dir="all", subs=None):
...
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:2984]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 3]
```diff
@@ -3085,7 +3085,7 @@ def plot_slope(self, dir="all", subs=None):
...
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3087]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 4]
```diff
@@ -3188,7 +3188,7 @@ def plot_deflection(self, dir="all", subs=None):
...
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3190]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 5]
```diff
@@ -3263,7 +3263,7 @@ def plot_loading_results(self, dir='x', subs=None):
...
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3265]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 6]
```diff
@@ -3360,7 +3360,7 @@ def plot_shear_stress(self, dir="all", subs=None):
         Examples
         ========
         There is a beam of length 20 meters and area of cross section 2 square
-        meters. It it supported by rollers at of its end. A linear load having
+        meters. It it supported by rollers at both of its ends. A linear load having
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3363]:
```suggestion
        meters. It is supported by rollers at both of its ends. A linear load having
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 7]
```diff
@@ -3456,7 +3456,7 @@ def max_shear_force(self):
         Examples
         ========
         There is a beam of length 20 meters. It it supported by rollers
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3458]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 8]
```diff
@@ -3531,7 +3531,7 @@ def max_bending_moment(self):
...
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3533]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 9]
```diff
@@ -3609,7 +3609,7 @@ def max_deflection(self):
...
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3611]:
```suggestion
        There is a beam of length 20 meters. It is supported by rollers
```

File: sympy/physics/continuum_mechanics/beam.py [hunk 10]
```diff
@@ -3641,3 +3641,4 @@ def max_deflection(self):
         max_def.append(self._max_deflection('y'))
         max_def.append(self._max_deflection('z'))
         return max_def
+
```
Comment (moorepants) [inline sympy/physics/continuum_mechanics/beam.py:3644]: Remove the extra line.

**Comment by moorepants** [comment]
Thanks for the fix. Can you fully correct the sentences and remove the extra line?

Also, this does not need a release notes, use "NO ENTRY".

**Comment by JebronLames32** [comment]
> Thanks for the fix. Can you fully correct the sentences and remove the extra line?
> 
> Also, this does not need a release notes, use "NO ENTRY".

Done. Please check now.

**Comment by moorepants** [comment]
You need to address this too:

```
This author is not included in the .mailmap file:
Abhinav R Cillanki <95158195+JebronLames32@users.noreply.github.com>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.

        For instructions on updating the .mailmap file see:
https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions

The following authors will be added to the AUTHORS file at the time of
the next SymPy release.
```

**Comment by JebronLames32** [comment]
Made the change @moorepants

**Comment by 1e9abhi1e10** [comment]
> Made the change @moorepants

you need to run the following command `python bin/mailmap_check.py` after adding your name and email in the `.mailmap` file 
read here for [instructions](https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions).

**Comment by JebronLames32** [comment]
> > Made the change @moorepants
> 
> you need to run the following command `python bin/mailmap_check.py` after adding your name and email in the `.mailmap` file read here for [instructions](https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions).

Thanks a lot

**Pull request merged**